### ISSUE TITLE
FR: Transparent window instead of screenshot

### ISSUE THREAD

**Issue by Meerkov** [SethRobinson/UGT#22]
Currently, when translating a window, the screen you look at is actually a screenshot with overlayed text.

Unfortunately, if the screen behind the translation changes, you will not be able to see any changes until you dismiss the translation. In the case of games, where characters may be walking or otherwise animated while dialog is on screen, this can be a minor annoyance. 

The same is true when translating intro-movies, where text appears for a few seconds while the intro movie plays (think scrolling text, like opening of Star Wars). The user may translate the text, but they can't see the movie, so they don't know when the next line of dialog is on screen.

**Comment by SethRobinson** [comment]
I agree this would be nice to have.  I [recently learned](https://stackoverflow.com/questions/4052940/how-to-make-an-opengl-rendering-context-with-transparent-background) that there are some GL flags that should be able to do this without resorting to GDI bitmap copying (like I do for the draggable rect) so maybe I can play around with that...
